src/Activity.js

```diff
@@ -1,58 +1,39 @@
-class Activity {
+import UserData from './UserData';
+
+class Activity extends UserData {
   constructor(activityData) {
+    super();
```

Comment:
Do we need to include anything in the super()?

Reply:
I am wondering the same thing about this.